lib/libutils/isoc/bget_malloc.c

```diff
@@ -181,6 +181,14 @@ static DEFINE_CTX(malloc_ctx);
 static __nex_data DEFINE_CTX(nex_malloc_ctx);
 #endif
 
+static void print_oom(size_t req_size __maybe_unused, void *ctx __maybe_unused)
+{
+#if defined(__KERNEL__) && defined(CFG_CORE_DUMP_OOM)
+	EMSG("mem alloc fail: sz %zu ctx %p", req_size, ctx);
```

Comment:
Minor: prefer "correct" syntax in `EMSG()` text. `"Memory allocation failed: size %zu context %p"`